avogadro/mainwindow.cpp

```diff
@@ -389,6 +389,8 @@ void MainWindow::setupInterface()
   if (pipeline) {
     pipeline->setAoEnabled(
       settings.value("MainWindow/ao_enabled", true).toBool());
+    pipeline->setDofEnabled(
+      settings.value("MainWindow/dof_enabled", true).toBool());
```

Comment:
Consider disabling DoF by default to optimize performance for lower-end systems. Users can enable it as needed via settings. Thoughts? @ghutchis

Reply:
this is still a work in progress PR and i want the shader file to be the best as needed and required. If you have any suggestions in this file for now https://github.com/OpenChemistry/avogadrolibs/pull/1662/files#diff-4f4a09755e6a3ad5776a815bd446b9ee8850fa794fea3aa67c1b1e2bd59e808e then please let me know. 

I plan to incorporate various user settings for enabling and disabling depth of field (DOF) once the .glsl file functions as intended. While it's currently functional, I aim to align with @ghutchis  plans for this feature.